### DISCUSSION TITLE
Evento Milhas ou Comprar Passagens

### DISCUSSION THREAD

**Discussion by bomffas** [frontendbr/forum#379] [Discussão]
Ae Galera,

Ano que vem pretendo ir em alguns eventos sobre a nossa area pelo Brasil, e gostaria de saber quem vai nesses eventos o que seria melhor, comprar a passagem ou usar as milhas ou assinar um plano de milhar como Smiles por exemplo.

**Comment by felipefialho** [comment]
Tenho usado bastante os Km do Posto Ipiranga para gerar milhas.

**Comment by kelciocajueiro** [comment]
Então, varia um pouco.

Dependendo do local de origem, destino e datas da viagem, as vezes compensa mais você comprar a passagem de fato e as vezes compensa mais usar milhas.

Quando eu preciso viajar faço o seguinte:

1. Vou no site das companhias aéreas e vejo como estão os preços.
2. Vou no site do MaxMilhas e vejo como estão os preços em milhas (O site te diz se pelas companhias aéreas está mais barato).
3. Faço a cotação com pessoas em grupo de milhas no Facebook.

No MaxMilhas o preço varia de acordo com o vendedor das milhas, você pode pagar parcelado no cartão mas também paga uma taxa do site (atualmente está em 40 reais). É confiável.

Em grupos de milhas no Facebook você tem que obter as referências do vendedor, que geralmente ficam em tópicos fixos no grupo. Tem que tomar cuidado porque tem muito picareta dando cano nos outros.

Para gerar milhas o melhor mesmo é o programa Km de vantagens do Posto Ipiranga. Com o dólar nas alturas não compensa de jeito nenhum com cartão de crédito.

**Comment by bomffas** [comment]
Ano que vem quero tentar ir no BrazilJS, FrontIn: BH POA Aracajú. Acredito que vai ser uma boa oportunidade para aprender.

**Comment by rubenmarcus** [comment]
@bomfimjr  eu to no club smiles.. e pago 150/mes pra 10000 milhas.. peguei no dia das crianças uma passagem ida e volta pra Belem por 20mil milhas..  deu 300 reais, a passagem normal custa uns 800 .. 

antes do cartão smiles eu usava o nubank.. que não tem nada em milhas.. o smiles apesar da anuidade e como sempre tenho fatura alta, gera algumas milhas.. poucas mas gera.. 

Acho que se você tiver a grana, compre a passagem na promoção, e gere mais milhas p vc.. no programa do club vc consegue viajar com as milhas pra fora 1x ao ano..  Ai se viajar tipo ,3,4x aqui nacional e ter uns gastos tipo 1000 reais ao mes pelo menos, acho que consegue mais passagens..

a vantagem é que as milhas demoram muitos anos pra expirar, então se você não usar em 1,2 anos pode durar até 2021 acredito eu..


essas promoções de passagens tem que ficar muito ligado, tem muita pegadinha e no mesmo dia a mesma passagem muda de preço muito rapido..

o voo que eu queria de manhã estava 15mil milhas, a meia noite no dia das crianças mas o site da smiles tava fora do ar pra fechar a compra.. ai de manhã o mesmo voo subiu pra 20mil milhas.. e consegui só pegar o voo da noite..


**Uma dica importante:** Não compre na decolar.com ... com medo de nao conseguir usar as milhas da Smiles, eu comprei no debito a passagem a 450 reais, que estava 900 antes.. ai cancelei depois (isso dia 13/10) 

Diziam que iam reembolsar em até 45 dias.. Me mandaram e-mail essa semana falando que foi depositado..

Nada..

ligo lá, dizem que não podem me atender..

decolar é uma empresa.. fd...

**Comment by yanmagale** [comment]
Aqui em BH tem uma startup,[Maxmilhas](https://www.maxmilhas.com.br/busca-passagens-aereas) , cujo produto é a venda de passagens aéreas através de milhas.

**Comment by marcusquiroga** [comment]
Em BH tem também a [Bankmilhas](https://bankmilhas.com.br) que te ajuda ao [vender milhas . ](https://bankmilhas.com.br/vender-milhas)

**Comment by felipefialho** [comment]
Estou juntando muitas milhas usando meu cartão de crédito (1 milha por 1 dolar) e os KM do Posto Ipiranga.

Inclusive consegui pagar uma passagem para o Chile dessa forma, no começo do ano.

Uso bastante.

**Comment by marcusquiroga** [comment]
Lfeh vc junta os KMs e transfere depois para os pontos da multiplus? Eu sempre que junto KMS(coloquei minha familia toda nisso, rs) transfiro depois para a multiplus e vendo as milhas. Acho que assim ao v[ender as milhas](https://bankmilhas.com.br) ganho muito mais.

**Comment by felipefialho** [comment]
Eu junto e depois transfiro. As vezes compensa fazer aquelas multiplicações doidas.

**Comment by xokxokxok** [comment]
Tem uma startup em SP tb chamada [p2pmilhas](https://www.p2pmilhas.com.br/), já usei é confiável.
